{"name": "Browse Our Categories Verify Functionality", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.StaleElementReferenceException: Message: stale element reference: element is not attached to the page document\n  (Session info: chrome=103.0.5060.114)\nStacktrace:\nBacktrace:\n\tOrdinal0 [0x006856B3+2184883]\n\tOrdinal0 [0x0061E5F1+1762801]\n\tOrdinal0 [0x00533DA8+802216]\n\tOrdinal0 [0x00536784+812932]\n\tOrdinal0 [0x00536642+812610]\n\tOrdinal0 [0x005368D0+813264]\n\tOrdinal0 [0x00563519+996633]\n\tOrdinal0 [0x005587F6+952310]\n\tOrdinal0 [0x0057CB0C+1100556]\n\tOrdinal0 [0x005582B4+950964]\n\tOrdinal0 [0x0057CD24+1101092]\n\tOrdinal0 [0x0058CF52+1167186]\n\tOrdinal0 [0x0057C926+1100070]\n\tOrdinal0 [0x00556EA0+945824]\n\tOrdinal0 [0x00557D96+949654]\n\tGetHandleVerifier [0x00924192+2704034]\n\tGetHandleVerifier [0x0091687D+2648461]\n\tGetHandleVerifier [0x0071119A+529066]\n\tGetHandleVerifier [0x00710006+524566]\n\tOrdinal0 [0x00624F2B+1789739]\n\tOrdinal0 [0x00629978+1808760]\n\tOrdinal0 [0x00629A65+1808997]\n\tOrdinal0 [0x00632A01+1845761]\n\tBaseThreadInitThunk [0x7694FA29+25]\n\tRtlGetAppContainerNamedObjectPath [0x77127A7E+286]\n\tRtlGetAppContainerNamedObjectPath [0x77127A4E+238]\n\n", "trace": "  File \"C:\\Users\\aaron\\Documents\\PythonProjects\\gettop-us\\venv\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Users\\aaron\\Documents\\PythonProjects\\gettop-us\\venv\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\verify_categories_steps.py\", line 52, in click_and_verify_category\n    clickable_ipad.click()\n  File \"C:\\Users\\aaron\\Documents\\PythonProjects\\gettop-us\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 88, in click\n    self._execute(Command.CLICK_ELEMENT)\n  File \"C:\\Users\\aaron\\Documents\\PythonProjects\\gettop-us\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 396, in _execute\n    return self._parent.execute(command, params)\n  File \"C:\\Users\\aaron\\Documents\\PythonProjects\\gettop-us\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 435, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\aaron\\Documents\\PythonProjects\\gettop-us\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 247, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "Given Gettop Home Page", "status": "passed", "start": 1658808202059, "stop": 1658808205204}, {"name": "When Browse Our Categories Text is Verified", "status": "passed", "start": 1658808205204, "stop": 1658808205627}, {"name": "And Verify 4 Correct Categories are Shown", "status": "passed", "start": 1658808205627, "stop": 1658808206490}, {"name": "Then Click Each Category and Verify the Correct Page Opened", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.StaleElementReferenceException: Message: stale element reference: element is not attached to the page document\n  (Session info: chrome=103.0.5060.114)\nStacktrace:\nBacktrace:\n\tOrdinal0 [0x006856B3+2184883]\n\tOrdinal0 [0x0061E5F1+1762801]\n\tOrdinal0 [0x00533DA8+802216]\n\tOrdinal0 [0x00536784+812932]\n\tOrdinal0 [0x00536642+812610]\n\tOrdinal0 [0x005368D0+813264]\n\tOrdinal0 [0x00563519+996633]\n\tOrdinal0 [0x005587F6+952310]\n\tOrdinal0 [0x0057CB0C+1100556]\n\tOrdinal0 [0x005582B4+950964]\n\tOrdinal0 [0x0057CD24+1101092]\n\tOrdinal0 [0x0058CF52+1167186]\n\tOrdinal0 [0x0057C926+1100070]\n\tOrdinal0 [0x00556EA0+945824]\n\tOrdinal0 [0x00557D96+949654]\n\tGetHandleVerifier [0x00924192+2704034]\n\tGetHandleVerifier [0x0091687D+2648461]\n\tGetHandleVerifier [0x0071119A+529066]\n\tGetHandleVerifier [0x00710006+524566]\n\tOrdinal0 [0x00624F2B+1789739]\n\tOrdinal0 [0x00629978+1808760]\n\tOrdinal0 [0x00629A65+1808997]\n\tOrdinal0 [0x00632A01+1845761]\n\tBaseThreadInitThunk [0x7694FA29+25]\n\tRtlGetAppContainerNamedObjectPath [0x77127A7E+286]\n\tRtlGetAppContainerNamedObjectPath [0x77127A4E+238]\n\n", "trace": "  File \"C:\\Users\\aaron\\Documents\\PythonProjects\\gettop-us\\venv\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Users\\aaron\\Documents\\PythonProjects\\gettop-us\\venv\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\verify_categories_steps.py\", line 52, in click_and_verify_category\n    clickable_ipad.click()\n  File \"C:\\Users\\aaron\\Documents\\PythonProjects\\gettop-us\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 88, in click\n    self._execute(Command.CLICK_ELEMENT)\n  File \"C:\\Users\\aaron\\Documents\\PythonProjects\\gettop-us\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 396, in _execute\n    return self._parent.execute(command, params)\n  File \"C:\\Users\\aaron\\Documents\\PythonProjects\\gettop-us\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 435, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\aaron\\Documents\\PythonProjects\\gettop-us\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 247, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1658808206490, "stop": 1658808213341}], "start": 1658808197311, "stop": 1658808214212, "uuid": "92388be5-53ef-4acf-9284-d79c07b0600e", "historyId": "68b4ac6c414f33a3707e7e1dc431d04c", "fullName": "Test Involving with Gettop.us Home Page: Browse Our Categories Verify Functionality", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "Test Involving with Gettop.us Home Page"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}